SonarQube.Old.Bootstrapper/SonarQube.Old.Bootstrapper.csproj

```diff
@@ -1,81 +1,34 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{35848C6E-2018-4831-94FA-40D7E45B5A09}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>SonarQube.Old.Bootstrapper</RootNamespace>
+    <TargetFramework>net46</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
At some point we should consider putting common properties in a _Directory.Build.props_ file so we only have to define them in one place.
Candidate properties: _TargetFramework_, _GenerateAssemblyInfo_, _DebugType_, and possibly even the _PackageReference_ for the analyzers. And _SonarLint.xml_ and the reference to the shared properties file.

Reply:
Also `SonarAnalyzer.CSharp` (although I am not sure whether SonarLint will handle it correctly).